models/recharge__monthly_recurring_revenue.sql

```diff
@@ -1,12 +1,12 @@
-with customers as (
-    select 
-        distinct date_month,
-        customer_id
-    from {{ ref('recharge__customer_daily_rollup') }}
-
-), billing as (
-    select *
-    from {{ ref('recharge__billing_history') }}
+with customers as ( 
+    select distinct  
+        date_month, 
+        customer_id 
+    from {{ ref('recharge__customer_daily_rollup') }} 
+ 
+), billing as ( 
+    select * 
+    from {{ ref('recharge__billing_history') }} 
```

Comment:
The customer daily rollup model is built off of the billing history model already. Can we just build exclusively off of the customer_daily_rollup and then just aggregate by month and add calculations on top of that? I think this would result in a more meaningful model and also a bit more informative regarding the MRR values. 

It reads a bit strange when one customer I looked up has two transactions from now canceled subscriptions and we are reporting their current MRR as a positive value. In this instance I would expect their MRR to be "0".

What are your thoughts?